functions.sh

```diff
@@ -19,6 +19,9 @@ function get_arch() {
 	s390x)
 		arch="s390x"
 		;;
+	arm64v8)
```

Comment:
Same here, you need to verify if the output `uname -m` is `arm64v8` or just `arm64` in arm64 device. We had some discussion about using `uname -m` here. If you run the scripts on an arm64 device, I think this would be fine. However, if you are using cross compiler then you may not able to get correct target platform here. And we need the capability to assign target platform if cross compiler is used.